src/main/flight/mixer_init.c

```diff
@@ -46,12 +48,29 @@
 
 PG_REGISTER_WITH_RESET_TEMPLATE(mixerConfig_t, mixerConfig, PG_MIXER_CONFIG, 0);
 
+#ifdef USE_RPM_LIMITER
+    #define RESET_MIXER_CONFIG_RPM_LIMITER_VALUES \
+    .rpm_limiter_p = 25,                   \
+    .rpm_limiter_i = 10,                   \
+    .rpm_limiter_d = 8,                    \
+    .rpm_limiter_rpm_limit = 0,            \
+    .motor_kv = 1960,
+#else
+    #define RESET_MIXER_CONFIG_RPM_LIMITER_VALUES
+#endif
+
+
+#define RESET_MIXER_CONFIG_VALUES   \
+    .mixerMode = DEFAULT_MIXER,     \
+    .yaw_motors_reversed = false,   \
+    .crashflip_motor_percent = 0,   \
+    .crashflip_expo = 35,           \
+    .mixer_type = MIXER_LEGACY,     \
+    RESET_MIXER_CONFIG_RPM_LIMITER_VALUES
+
+
 PG_RESET_TEMPLATE(mixerConfig_t, mixerConfig,
-    .mixerMode = DEFAULT_MIXER,
-    .yaw_motors_reversed = false,
-    .crashflip_motor_percent = 0,
-    .crashflip_expo = 35,
-    .mixer_type = MIXER_LEGACY,
+    RESET_MIXER_CONFIG_VALUES
```

Comment:
@KarateBrot  @blckmn 
i made this weird trick here, but it needs a review, maybe there is a better idea....